datalad/support/annexrepo.py

```diff
@@ -748,13 +748,18 @@ def get_remotes(self,
         else:
             return remotes
 
-    def get_special_remotes(self):
+    def get_special_remotes(self, include_dead:bool =False) -> dict[str, dict]:
```

Comment:
Well, space doesn't have any functional meaning here and just styling which we never enforced. Not sure which one is the most kosher way since we do not add spaces around `=` for keyword args

<details>
<summary>But we seems generally (not always) add at the type annotated places</summary> 

```shell
❯ git grep 'def .*:.* ='
datalad/runner/exception.py:    def to_str(self, include_output: bool = True) -> str:
datalad/runner/protocol.py:    def __init__(self, done_future: Any = None, encoding: Optional[str] = None) -> None:
datalad/runner/tests/test_nonasyncrunner.py:def test_popen_invocation(src_path: str = "", dest_path: str = "") -> None:
datalad/runner/tests/test_witless_runner.py:def test_runner(tempfile: str = "") -> None:
datalad/runner/tests/test_witless_runner.py:def test_runner_failure(dir_: str = "") -> None:
datalad/runner/tests/test_witless_runner.py:def test_runner_fix_PWD(path: str = "") -> None:
datalad/runner/tests/test_witless_runner.py:def test_runner_cwd_encoding(path: str = "") -> None:
datalad/runner/tests/test_witless_runner.py:def test_runner_stdin(path: str = "") -> None:
datalad/runner/tests/test_witless_runner.py:def test_asyncio_loop_noninterference1(path1: str = "", path2: str = "") -> None:
datalad/runner/tests/test_witless_runner.py:def test_asyncio_forked(temp_: str = "") -> None:
datalad/runner/tests/test_witless_runner.py:def test_environment(temp_dir_path: str = "") -> None:
datalad/support/ansi_colors.py:def format_msg(fmt: str, use_color: bool = False) -> str:
datalad/support/ansi_colors.py:def color_word(s: str, color: Optional[int], force: bool = False) -> str:
datalad/support/cookies.py:    def __init__(self, filename: Optional[str | Path] = None) -> None:
datalad/support/digests.py:    def __init__(self, digests: Optional[list[str]] = None, blocksize: int = 1 << 16) -> None:
datalad/support/globbedpaths.py:    def __init__(self, patterns: Optional[Iterable[str | bytes]], pwd: Optional[str] = None, expand: bool = False) -> None:
datalad/support/globbedpaths.py:    def expand(self, full: bool = False, dot: bool = True, refresh: bool = False,
datalad/support/globbedpaths.py:    def expand_strict(self, full: bool = False, dot: bool = True, refresh: bool = False) -> list[str]:
datalad/support/network.py:def quote_path(path: str, safe: str = "/") -> str:
datalad/support/path.py:def get_parent_paths(paths: list[str], parents: list[str], only_with_parents: bool = False, *, sep: str = '/') -> list[str]:
datalad/utils.py:def find_files(regex: str, topdir: str | Path = curdir, exclude: Optional[str]=None, exclude_vcs: bool =True, exclude_datalad: bool =False, dirs: bool =False) -> Iterator[str]:
datalad/utils.py:def expandpath(path: str | Path, force_absolute: bool =True) -> str:
datalad/utils.py:def rotree(path: str | Path, ro: bool =True, chmod_files: bool =True) -> None:
datalad/utils.py:def rmtree(path: str | Path, chmod_files: bool | Literal["auto"] ='auto', children_only: bool =False, *args: Any, **kwargs: Any) -> None:
datalad/utils.py:def get_open_files(path: str | Path, log_open: int = False) -> dict[str, Any]:
datalad/utils.py:def file_basename(name: str | Path, return_ext: Literal[False] = False) -> str:
datalad/utils.py:def file_basename(name: str | Path, return_ext: bool =False) -> str | tuple[str, str]:
datalad/utils.py:def ensure_dict_from_str(s: str, sep: str = '\n') -> Optional[dict[str, str]]:
datalad/utils.py:def ensure_dict_from_str(s: dict[K, V], sep: str = '\n') -> Optional[dict[K, V]]:
datalad/utils.py:def ensure_dict_from_str(s: str | dict[K, V], sep: str = '\n') -> Optional[dict[str, str]] | Optional[dict[K, V]]:
datalad/utils.py:def generate_file_chunks(files: list[str], cmd: str | list[str] | None = None) -> Iterator[list[str]]:
datalad/utils.py:def __auto_repr__(obj: Any, short: bool =True) -> str:
datalad/utils.py:    def fake_print(*args: str, sep: str = ' ', end: str = "\n", file: Optional[IO[str]] = None) -> None:
datalad/utils.py:    def assert_logged(self, msg: Optional[str]=None, level: Optional[str]=None, regex: bool =True, **kwargs: Any) -> None:
datalad/utils.py:def swallow_logs(new_level: str | int | None = None, file_ : str | Path | None = None, name: str='datalad') -> Iterator[SwallowLogsAdapter]:
datalad/utils.py:def dlabspath(path: str | Path, norm: bool =False) -> str:
datalad/utils.py:def make_tempfile(content: str | bytes | None = None, wrapped: Optional[Callable[..., Any]] = None, **tkwargs: Any) -> Iterator[str]:
datalad/utils.py:def read_file(fname: str | Path, decode: Literal[True] =True) -> str:
datalad/utils.py:def read_file(fname: str | Path, decode: Literal[True, False] =True) -> str | bytes:
datalad/utils.py:def read_csv_lines(fname: str | Path, dialect: Optional[str] = None, readahead: int=16384, **kwargs: Any) -> Iterator[dict[str, str]]:
datalad/utils.py:    def __init__(self, separator: str=" ", element_formatter: string.Formatter =string.Formatter(),
datalad/utils.py:def create_tree(path: str, tree: TreeSpec, archives_leading_dir: bool =True, remove_existing: bool =False) -> None:
datalad/utils.py:def bytes2human(n: int | float, format: str ='%(value).1f %(symbol)sB') -> str:

```
</details>

so I will accept this suggestion, thank you!